### ISSUE TITLE
AUR Package Can't Download Launcher

### ISSUE THREAD

**Issue by Lynnesbian** [CodingKoopa/lucas-simpsons-hit-and-run-mod-launcher-linux-launcher#6]
The AUR package fails to install with the following error:
```
==> ERROR: Failure while downloading https://donutteam.com/apps/Lucas'%20Simpsons%20Hit%20&%20Run%20Mod%20Launcher%201.18.zip
    Aborting...
```

When downloading the launcher manually from donutteam.com, the URL used is `https://cdn.donutteam.com/Downloads/Tools/Lucas'%20Simpsons%20Hit%20&%20Run%20Mod%20Launcher%201.19.zip`. The AUR package should be updated to reflect this new URL format.

**Comment by Lynnesbian** [comment]
It's also worth noting that the AUR package specifies version `1.18`, which (as far as I could tell) isn't available from the website anymore. The current version is `1.19`.

**Comment by CodingKoopa** [comment]
Thanks for the report. Should be fixed by [`1165024f9875bb4fe6fc0adf616153f9f57322b8`](https://aur.archlinux.org/cgit/aur.git/commit/?h=lucas-simpsons-hit-and-run-mod-launcher&id=1165024f9875bb4fe6fc0adf616153f9f57322b8).
